### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by malinkb**
Updated a more automatic way to add the sources and keys and added click to copy commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,24 @@ Discord APT repository
...
 --
-1. Create a file `/etc/apt/sources.list.d/discord.list` with the contents `deb https://palfrey.github.io/discord-apt/debian/ ./`
-2. Download the file https://palfrey.github.io/discord-apt/discord-apt.gpg.asc to `/etc/apt/trusted.gpg.d`
-3. `sudo apt-get update`
-4. `sudo apt-get install discord`
+1. Import discord-apt signing key:
+##
+    `wget -q https://palfrey.github.io/discord-apt/discord-apt.gpg.asc -O- | sudo tee /etc/apt/keyrings/discord-apt.gpg.asc > /dev/null`
+
+3. The fingerprint should be `2D3E44DC23D20638A854659F8F8F44E1B741DD1E`. You may check it with the following command:
```
Comment (palfrey) [inline README.md:12]: 1 then 3?

File: README.md [hunk 2]
```diff
@@ -3,12 +3,24 @@ Discord APT repository
...
+##
+    `gpg -n -q --import --import-options import-show /etc/apt/keyrings/packages.mozilla.org.asc | awk '/pub/{getline; gsub(/^ +| +$/,""); if($0 == "2D3E44DC23D20638A854659F8F8F44E1B741DD1E") print "\nThe key fingerprint matches ("$0").\n"; else print "\nVerification failed: the fingerprint ("$0") does not match the expected one.\n"}`
```
Comment (palfrey) [inline README.md:14]: Why is `/etc/apt/keyrings/packages.mozilla.org.asc` here? Also, this command doesn't work on zsh for one. Is there a much less complicated form that just shows the fingerprint? Maybe the first gpg command?

File: README.md [hunk 3]
```diff
@@ -3,12 +3,24 @@ Discord APT repository
...
+
+4. Add the Discord APT repository to your sources list:
+##
+    `echo "deb https://palfrey.github.io/discord-apt/debian/ ./" | sudo tee -a /etc/apt/sources.list.d/discord.list > /dev/null`
+
+6. Update package list and install the discord .deb package
```
Comment (palfrey) [inline README.md:20]: 4 then 6?

**Review by palfrey — CHANGES_REQUESTED** [review]
image

I'm not entirely convinced by this so far. It's got weird formatting (see image), the commands have backticks around them which means the copy/pasting doesn't work, and the numbering is broken.

If you can fix all of that, I might be interested in this, but TBH I'm happy enough with the current state of things and if people are having difficulty figuring out what to do, I'm actually pro that, as I don't really want to be doing customer support for folks who can't figure it out from the current instructions. That's the sort of thing Discord themselves should be doing!

File: README.md (hunk 3 above)
Comment (malinkb) [reply README.md]: I've fixed the numbering

File: README.md (hunk 2 above)
Comment (malinkb) [reply README.md]: I used this site as an inspiration: https://support.mozilla.org/en-US/kb/install-firefox-linux#w_install-firefox-deb-package-for-debian-based-distributions

I fixed the path so it matches the one for discord

File: README.md (hunk 1 above)
Comment (malinkb) [reply README.md]: I've fixed the quirks.
I agree discord itself should've have a debian APT repo though. 
Even Microsoft have their own Debian APT repos

File: README.md [hunk 4]
```diff
@@ -3,12 +3,24 @@ Discord APT repository
...
 
-Usage instructions
+## Usage instructions
 --
-1. Create a file `/etc/apt/sources.list.d/discord.list` with the contents `deb https://palfrey.github.io/discord-apt/debian/ ./`
-2. Download the file https://palfrey.github.io/discord-apt/discord-apt.gpg.asc to `/etc/apt/trusted.gpg.d`
-3. `sudo apt-get update`
-4. `sudo apt-get install discord`
+1. Import discord-apt signing key:
+##
```
Comment (fey) [inline README.md:9]: Do not use empty `##` tags. Its for level 2 headers.

You can format code block with triple backticks and `bash` language instead using indent. [How it works](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)

**Comment by fey** [comment]
I checked the commands, they work. I have left comments where I need to correct them.

**Comment by palfrey** [comment]
I've been thinking about this a bit, and realised I have no interest in things like this. Please see https://github.com/palfrey/discord-apt/commit/514920d2cde112a601672b867f79f2fd4d58e850. Sorry about this.

**Pull request closed without merging**